lib/response-builder.ts

```diff
@@ -77,6 +77,14 @@ async function buildResponse(
     additionalOsDepsFacts.push(imageLabels);
   }
 
+  if (depsAnalysis.imageCreationTime) {
+    const imageCreationTimeFact: facts.ImageCreationTimeFact = {
+      type: "imageCreationTime",
+      data: depsAnalysis.imageCreationTime,
+    };
+    additionalOsDepsFacts.push(imageCreationTimeFact);
```

Comment:
Does it make sense to add it to this array of facts (it's not really an os-deps fact, but I see that almost all facts are added to this array). Or maybe this array can be renamed to additionalFacts?

Reply:
Yes, it looks like this array has been abused 😆 If you fancy renaming that would be awesome 🙇‍♀️